tests/test_project/models.py

```diff
@@ -43,3 +43,22 @@ class RadiusAccounting(models.Model):
                                 db_index=True,
                                 null=True,
                                 blank=True)
+
+
+class Id(models.Model):
```

Comment:
please rename this model to something a bit more meaningful, for example `Project`